assets/app/styles/_core.less

```diff
@@ -860,6 +860,47 @@ a.disabled-link {
   }
 }
 
+.dockerfile-editor {
+  min-height: 200px;
+}
+
+.edit-form {
+  margin: 30px auto;
+  .form-groups {
+    padding-left: 10px;
+    border-left: 1px solid #eee;
+  }
+  .form-groups:hover {
+    padding-left: 10px;
+    border-left: 1px solid #0099d3;
+  }
+  .labels {
+    .form-group {
+      width: 44%;
+      .form-control {
+        width: 100%;
+      }
+    }
+  }
+  .checkbox {
+    margin: 10px;
+  }
+  .section {
+    padding-bottom: 15px;
+  }
+  .warnings {
+    text-align: center;
```

Comment:
Is centering working? It looks like the text is left-aligned (which I'm fine with).